.github/workflows/pr.yml

```diff
@@ -80,7 +80,7 @@ jobs:
       run: gradle clean build
 
     - name: Run integration tests
-      run: gradle integrationTest
+      run: gradle integrationTest --info
```

Comment:
I'd prefer to leave this level of logging on for now so we can root cause test failures with a bit more ease.

Reply:
I don't think running the integration suite locally is a good substitute for debugging problems that happen in CI, and given that a lot of the errors we see are transitory and are not likely to be immediately reproducible by re-running locally, I think logging additional information as a part of CI is important.

Reply:
What's the downside of enabling logging for CI runs? Do we pay more?